### DISCUSSION TITLE
NES roms don't seem to work

### DISCUSSION THREAD

**Discussion by davidcsz** [eduardofilo/RG350_adam_image#94] [General]
Attempting to launch any NES games takes me to the loading screen, but ends up back at SimplMenu. Tried all available retroarch cores and the standalone.

**Comment by eduardofilo** [comment]
I have no problem with NES games, so it should be ROM related. Try to unzip the ROMs if they are zipped.
For solving this cases, always help to view the RetroArch log. Try this steps:

1. Disable SimpleMenu as default frontend.
2. In GMenu2X, press Start and enable `Output logs`.
3. Open the NES emulator through the GMenu2X launcher in `retroarch` section.
4. You will surely return to GMenu2X.
5. Go to `settings` section and open Log viewer app.

Tell me what you see there.

If you can open a SSH/SFTP session, better download the file `/media/data/local/home/.gmenu2x/gmenu2x.log` and upload here to analyze it.

**Reply by chocolatelady** [reply]
Reviving this thread as I'm having the same problem, here's what I got on the log viewer. If it helps, I'm able to run these ROMs on my Retroid Pocket 3+ using the Mesen core, so not sure what I'm doing wrong here!

![NES 2](https://user-images.githubusercontent.com/122855417/212818856-8ace8752-83b6-4671-9770-8215f040af0c.jpg)

![NES 1](https://user-images.githubusercontent.com/122855417/212818814-42ced1d6-7e6b-442b-9428-5fe648e277cd.jpg)

**Comment by davidcsz** [comment]
I think you're correct. It was the ROMs. Thanks!
